js/src/modals/Transfer/Details/details.js

```diff
@@ -88,7 +107,12 @@ export default class Details extends Component {
           <div>
             <Input
               disabled
-              label='total transaction amount'
+              label={
+                <FormattedMessage
+                  id='transfer.details.total/label'
```

Comment:
Damn keyboard. Thank you, I didn't just miss it while typing, also missed it when reading through.